Testscripts/Linux/xfstesting.sh

```diff
@@ -237,15 +251,16 @@ Main() {
             # Azure provides links in /dev/disk/azure/ to /dev/sd[a-z] devices
             # this is needed as some 5+ kernels do not assign drive letters in any particular order
             # (previously the test disk was /dev/sdc)
-            testDisk=$(readlink -f /dev/disk/azure/scsi1/* | sed "s@/dev/@@g" | head -1)
+            #testDisk=$(readlink -f /dev/disk/azure/scsi1/* | sed "s@/dev/@@g" | head -1)
+            testDisk="sdc"
```

Comment:
we can't hard code the disk name, it may change.

Reply:
Reverted to previous commit.